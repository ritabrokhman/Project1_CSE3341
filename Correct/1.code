
123
      and procedure object OBJECT
      (
        ;
        =

        token
        ==
      )
3
while