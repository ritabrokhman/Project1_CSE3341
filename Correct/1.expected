CONST[123]
AND
PROCEDURE
OBJECT
ID[OBJECT]
LPAREN
SEMICOLON
ASSIGN
ID[token]
EQUAL
RPAREN
CONST[3]
ID[while]
